### ISSUE TITLE
Create Exe or Debian Package

### ISSUE THREAD

**Issue by dhavalsolanki** [edvin/fxlauncher#178]
Hello,
I am using fxlauncher in my application. Is there any way to create exe and debian package using fxlauncher api.

**Comment by edvin** [comment]
Yes, that's literally what it does - package your app and wrap it in an executable. Please see one of the example projects (maven or gradle).

**Comment by dhavalsolanki** [comment]
But it is actually generating executable exe. Can i generate installer for my project ?

**Comment by edvin** [comment]
Yes, there is a generate-installer step in the example projects which builds installers. On windows, it has a prerequisite on a third party installer framework, which it will prompt you for during installation.

**Comment by dhavalsolanki** [comment]
Hello,
Thanks for the response.
Now i am stuck in creating a debian package of fxlauncher, is there any way to create it using any maven plugin

**Comment by edvin** [comment]
Fxlauncher uses javapackager to create a deb package if you include `-native deb`, like it is described in the docs:

https://docs.oracle.com/javase/8/docs/technotes/tools/unix/javapackager.html
